scripts/exregional_run_post.sh

```diff
@@ -188,12 +188,15 @@ post_dd=${post_time:6:2}
 post_hh=${post_time:8:2}
 
 cat > itag <<EOF
-${dyn_file}
-netcdf
-grib2
-${post_yyyy}-${post_mm}-${post_dd}_${post_hh}:00:00
-${POST_FULL_MODEL_NAME}
-${phy_file}
+&model_inputs
+fileName='${dyn_file}'
+IOFORM='netcdf'
+grib='grib2'
+DateStr='${post_yyyy}-${post_mm}-${post_dd}_${post_hh}:00:00'
+MODELNAME='${POST_FULL_MODEL_NAME}'
+fileNameFlux='${phy_file}'
+fileNameFlat='postxconfig-NT.txt'
+/
```

Comment:
It would be good if the content (i.e. Line 192-198) inside the "model_inputs" namelist is indented.

Reply:
Thanks Guoqing.  I made that change...I'm assuming it won't change how the lines are read in?